### DISCUSSION TITLE
Is there a way to undo data changes on the phone?

### DISCUSSION THREAD

**Discussion by ncsumaize** [PhenoApps/Field-Book#728] [Q&A] [answered]
If someone overwrites data on the phone, it seems to get saved immediately. Is there a way to undo this directly on the phone to avoid database conflicts that have to be resolved later?

**Comment by trife** [comment, marked as answer]
@ncsumaize Recording a phenotype is similar to to editing a cell in Excel- there isn't a history tied to the individual cell. You can [lock](https://docs.fieldbook.phenoapps.org/en/latest/collect.html#top-toolbar) data input to restrict editing existing entries; once you navigate away from the current "cell" the data is locked for the user.
